matlab/func/cpm_cv.m

```diff
@@ -7,27 +7,26 @@
 % y_test       y data used for testing
 % y_predict    Predictions of y data used for testing
 
-% Split data
+% cross-validation indices
 nsubs=size(x,2);
-randinds=randperm(nsubs);
```

Comment:
May I ask why this line has been deleted? It seems that without permutation of subjects, each iteration of CPM will yield identical results. This would defeat the intention to generate a empirical distribution of model performances.